src/modules/flow/http-client/http-client.json

```diff
@@ -777,6 +777,161 @@
       ]
     },
     {
+      "name": "http-client/create-url",
+      "private_data_type": "create_url_data",
+      "category": "http-generic",
+      "description": "Creates an encoded URL",
+      "url": "http://solettaproject.org/doc/latest/node_types/http-client/create-url.html",
+      "methods": {
+        "close": "create_url_close",
+        "open": "create_url_open"
+      },
+      "options": {
+        "members": [
+          {
+            "data_type": "string",
+            "default": "http",
+            "description": "The URL protocol",
+            "name": "scheme"
+          },
+          {
+            "data_type": "int",
+            "default": 80,
+            "description": "The server port",
+            "name": "port"
+          },
+          {
+            "data_type": "string",
+            "default": null,
+            "description": "The server to connect with",
+            "name": "host"
+          },
+          {
+            "data_type": "string",
+            "default": "/",
+            "description": "The serve path",
+            "name": "path"
+          },
+          {
+            "data_type": "string",
+            "default": null,
+            "description": "The URL fragment",
+            "name": "fragment"
+          },
+          {
+            "data_type": "string",
+            "default": null,
+            "description": "The URL parameters to be encoded",
```

Comment:
say it should be already encoded in the proper format (ie: a=b&c=%30). The message says it wil be encoded, which is wrong.